config/routes.rb

```diff
@@ -1,8 +1,7 @@
 # frozen_string_literal: true
 
-MyPluginModule::Engine.routes.draw do
-  get "/examples" => "examples#index"
-  # define routes here
-end
+DiscourseTopicNoindex::Engine.routes.draw {}
```

Comment:
Since there are no routes actually being defined, I think you could just delete this whole file?